tasks/redhat.yml

```diff
@@ -19,13 +19,14 @@
     path: "/etc/sysconfig/network-scripts/ifcfg-{{ item }}"
     state: absent
   when:
-    - item not in ansible_facts.interfaces
```

Comment:
I have seen a case where this was useful. Cloud image provided an invalid ens3. I had ens3, but did not want to configure it explicitly - only VLAN interfaces.

Reply:
the simple production case: network pxe (native VLAN), network mgmt (VLAN11) - provision creates the ifcfg-ens3 without IP and ifcfg-ens3.11 with DHCP IP from ip-allocation. controllers/compute configuration have the bond0 with ens3 and ens4 interfaces, and the same management VLAN11 and IP. overcloud host configure creates ifcfg-ens3 and ifcfg-ens4 as slaves, ifcfg-bond0 and ifcfg-bond0.11 with management IP. BUT there are stale ifcfg-ens3.11 still exists on filesystem with DHCP and second default route via it.

Reply:
there is the same issue with systemd-networkd configuration on Ubuntu. the solution will provide in the gerrit review later.

Reply:
I have seen that issue before, and this is a reasonable solution for it.